### PR TITLE
fix issue #102

### PR DESCRIPTION
**Pull request by alhelwany**

### CODE REVIEW

File: src/app/(auth)/components/signin-form.tsx [hunk 1]
```diff
@@ -21,83 +21,95 @@ export const SignInForm = () => {
...
-          alt="Facebook icon"
-        />
-        <Image
-          src="/images/google-icon.png"
-          width={64}
-          height={64}
-          alt="Google icon"
-        />
+      <div className="mx-20 my-10">
+        <a href="/signup">
```
Comment (coltonehrman) [inline src/app/(auth)/components/signin-form.tsx:107]: Please use Next.js `Link` components for links.

Comment (coltonehrman) [reply src/app/(auth)/components/signin-form.tsx:107]: https://nextjs.org/docs/app/api-reference/components/link

**Comment by coltonehrman** [comment]
@alhelwany please format your code

**Comment by coltonehrman** [comment]
@all-contributors please add @alhelwany for code

**Comment by coltonehrman** [comment]
@alhelwany Please fix the build error in CI

**Comment by coltonehrman** [comment]
@alhelwany Can you include a screenshot of the new UIs please?

**Comment by coltonehrman** [comment]
@all-contributors please add @alhelwany for code

**Comment by alhelwany** [comment]
Screenshot from 2024-04-22 22-03-36
Screenshot from 2024-04-22 22-03-25

**Pull request merged**